book/tutorials/data-access-and-format/earthaccess.ipynb

```diff
@@ -1117,10 +1117,10 @@
     "\n",
     "Direct-access to data from an S3 bucket is a two step process.  First, the files are opened using the `open` method.  This first step creates a Python file-like object that is used to load the data in the second step.  \n",
     "\n",
-    "Authentication is required for this step.  The `auth` object created at the start of the notebook is used to provide Earthdata Login authentication and AWS credentials \"_behind-the-scenes_\".\n",
+    "Authentication is required for this step.  The `auth` object created at the start of the notebook is used to provide Earthdata Login authentication and AWS credentials \"_behind-the-scenes_\". These credentials expire after one hour so the `auth` object must be executed within that time window prior to these next steps. \n",
```

Comment:
@betolink Can you confirm this.  I thought auth automatically re-authenticated.

Reply:
Ah, I thought this would have to be re-run if you happen to execute auth more than an hour prior to running the `open` method. That's great if that's not the case!